base/applications/osk/lang/it-IT.rc

```diff
@@ -1,129 +1,129 @@
 /*
- * PROJECT:         ReactOS Kernel
- * LICENSE:         GPL - See COPYING in the top level directory
- * FILE:            base/applications/osk/lang/it-IT.rc
- * PURPOSE:         On screen keyboard (Italian resources)
- * PROGRAMMERS:     Denis ROBERT
+ * PROJECT:     ReactOS Virtual Keyboard - OSK
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     Virtual Keyboard On screen (Italian resources)
+ * COPYRIGHT:   Copyright Denis ROBERT
```

Comment:
You changed the license, so you need to ask all authors if this is ok.

Reply:
And what do you do when the author is deceased/left without any traces/etc...?
Also, the COPYING in top-level directory looks like GPL-2 (with our addition too).